broker_test.go

```diff
@@ -826,6 +835,7 @@ func (s *BrokerSuite) TestPartitionOffset(c *C) {
 						{
 							ID:      1,
 							Offsets: []int64{123, 0},
+							Err:     resErr,
```

Comment:
Thanks for adding this test!  It looks like the errors you're returning in the test are in `part.Err`, not as a result of the overall `Offset` call.